### PR TITLE
Code optimalisations

### PR DESCRIPTION
**Pull request by poespas**
Hey, I'm remaking this PR. Make sure you test your changes locally, and clear your cache.

For any questions, leave a comment! :)

### CODE REVIEW

**Comment by nm17** [comment]
Good PR

**Comment by s-crypt** [comment]
This breaks:

- The big counter number
- The debt added section no longer exists

File: index.html [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
+				</ul>
+			</div>
+			<div class="added">
+				<div class="lh">
+					DEBT ADDED
+				</div>
+				<ul>
+					<li>$2,399,727,315 the worst <a href="https://youtu.be/BQaSlD_UFRU?t=10675">Deal or No Deal</a> play of all time</li>
+					<li>$250,000,000 <a href="https://youtu.be/BQaSlD_UFRU?t=7300">Jerma bets 250 million that he hasn't said "GAS" more than the movie's script.</a> He failed the bet.</li>
+					<li>$1,000,000,000 <a href="https://youtu.be/5RL0ZY2xpZs?t=6431">Jerma bets $1 billion that he will finish The Impossible Quiz.</a> He failed the bet.</li>
```
Comment (Thestickman391) [inline index.html:56]: This has been audited and is inccorect

**Review by s-crypt — APPROVED** [review]
fixed the impossible quiz clerical error

**Comment by s-crypt** [comment]
It broke in deployment

**Pull request merged**